sonarscanner-templates/laravel.properties

```diff
@@ -0,0 +1,34 @@
+# For additional SonarScanner analysis parameters.
+# https://docs.sonarqube.org/latest/analysis/analysis-parameters/
+
+# --- mandatory properties ---
+
+# Must be unique in a given SonarQube instance.
+sonar.projectKey=
+
+# The SonarQube server URL.
+sonar.host.url=http://sonarqube:9000
+
+# The authentication token or login of a SonarQube user with
+# Execute Analysis permission on the project.
+sonar.login=
+
+# --- optional properties ---
+
+# Defaults to project key.
+sonar.projectName=
+
+# Defaults to 'not provided'.
+# sonar.projectVersion=
+
+# Path is relative to the sonar-project.properties file. Defaults to .
+sonar.sources=app,config,resources/views,routes
```

Comment:
I would recommend including the database folder, since we have model factories, database seeders and migrations. This would allow SonarQube to analyse code that might cause vulnerabilities to your database layer.